Grand Circus - OOP/GrandCircus/CircusModel/Circus.cs

```diff
@@ -0,0 +1,30 @@
+﻿using iQuest.GrandCircus.Presentation;
+using System.Collections.Generic;
+
+namespace iQuest.GrandCircus.CircusModel
+{
+    internal class Circus
+    {
+        List<IAnimal> animals=new List<IAnimal>();
+        public Circus(Arena arena)
+        {
+            animals.Add(new Lion("Mufasa"));
+            animals.Add(new Bear("Baloo"));
+            animals.Add(new Snake("Ka"));
+            animals.Add(new Elephant("Admiral")); 
+            
+        }
+
+        public void Perform()
+        {
+            Arena arena=new Arena();
```

Comment:
We already created an object for 'Arena' in Program.cs. We need to use that object instead of creating a new one.
Hint: use the parameter received on the constructor.

Reply:
Not sure what is the best way to approach this, I declared a new variable to hold the parameter from the constructor, but I'm still not sure it's the best approach